web/Controllers/AdminController.cs

```diff
@@ -4,6 +4,7 @@
 using BricksAndHearts.Services;
 using BricksAndHearts.ViewModels;
 using LINQtoCSV;
+using Microsoft.ApplicationInsights.AspNetCore.Extensions;

```

Comment:
Is this import used?

Reply:
I dont think so! I feel like theres quite a few unused imports across lots of the files, maybe it's worth raising a ticket to try and get rid of them?

Reply:
Rider has an organise imports action, feel free to just do it and raise a PR